scripts/OverlaySurvey.py

```diff
@@ -52,8 +46,13 @@ def update_results(graph, parent_info, parent_key, results, is_inbound):
         results[direction_tag][other_key] = peer
         add_new_node(graph, parent_key)
         add_new_node(graph, other_key, peer["version"])
-        add_new_edge(graph, other_key, parent_key,
-                     peer["bytesWritten"] + peer["bytesRead"])
+        edge_properties = peer.copy()
```

Comment:
the old code used to no-op if it was a duplicate connection, this calls `add_edge` again:
* I think it's possible to get duplicate connection information if we survey the same node several times (because we don't have enough connections for example)
* it looks like `add_edge` performs an update already so codewise, I think we're good

we should add a comment to call out what we're really doing here

Reply:
added a comment!